libfreerdp/core/test/TestSettings.c

```diff
@@ -1382,6 +1382,106 @@ static BOOL test_validity_check(void)
 	return log_result(rc, __func__);
 }
 
+static BOOL test_string_null(rdpSettings* settings, FreeRDP_Settings_Keys_String id)
+{
+	if (!freerdp_settings_set_string(settings, id, NULL))
+		return FALSE;
+
+	const char* chk = freerdp_settings_get_string(settings, id);
+	return (chk == NULL);
+}
+
+static BOOL test_string_check(rdpSettings* settings, FreeRDP_Settings_Keys_String id,
```

Comment:
Maybe something like `test_string_equal()` would be more precise as a name here?

Reply:
true, but not really relevant.